drivers/volume/portworx/portworx.go

```diff
@@ -99,8 +105,13 @@ const (
 
 	validateSnapshotTimeout       = 5 * time.Minute
 	validateSnapshotRetryInterval = 10 * time.Second
+
+	pxEnableTLS    = "PX_ENABLE_TLS"
+	pxSharedSecret = "PX_SHARED_SECRET"
 )
 
+var endpointPort string
```

Comment:
Move this to the `portworx` struct. Make all the methods accessing this variable members of the struct too